### PR TITLE
IA-1738: update readme.md

### PR DESCRIPTION
**Pull request by jsmigovsky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,12 +39,19 @@ A barebones deployment that results in a task that runs every 7 days.
...
 module "simple-task" {
-  source              = "USSBA/easy-fargate/aws"
-  version             = "~> 2.0"
-  name                = "my-simple-task"
-  container_image     = "ubuntu:latest"
-  container_command   = ["curl", "https://www.google.com"]
-  schedule_expression = "rate(7 days)"
+  #source              = "USSBA/easy-fargate/aws"
+  #version             = "~> 3.0"
+  source = "../../"
```
Comment (elijahmuha) [inline README.md:44]: uncomment the top two lines and remove `source = "../../"`

File: README.md [hunk 2]
```diff
@@ -86,18 +122,6 @@ module "my-fargate-task" {
...
-      name  = "FOO"
-      value = "bar"
-    }
-  ]
-  container_secrets = [
-    {
-      name      = "FOO_SECRET"
-      valueFrom = "arn:aws:ssm:${local.region}:${local.account_id}:parameter/foo_secret"
-    }
-  ]
```
Comment (elijahmuha) [inline README.md:100]: I kinda like keeping the environment variables in the example for some reason but I guess it's not necessary?

**Review by elijahmuha — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jsmigovsky) [reply README.md:100]: these properties do not exist anymore, they are now part of the container definition itself.

Comment (elijahmuha) [reply README.md:100]: Great reason to not that them in there

File: README.md [hunk 3]
```diff
@@ -40,11 +40,18 @@ A barebones deployment that results in a task that runs every 7 days.
 ```terraform
 module "simple-task" {
   source              = "USSBA/easy-fargate/aws"
-  version             = "~> 2.0"
-  name                = "my-simple-task"
-  container_image     = "ubuntu:latest"
-  container_command   = ["curl", "https://www.google.com"]
-  schedule_expression = "rate(7 days)"
+  version             = "~> 3.0"
+  source = "../../"
```
Comment (elijahmuha) [inline README.md:44]: still need to remove this line

**Review by elijahmuha — CHANGES_REQUESTED** [review]

**Review by elijahmuha — APPROVED** [review]

**Pull request merged**